src/gui/accountstate.cpp

```diff
@@ -205,6 +222,16 @@ void AccountState::setNavigationAppsEtagResponseHeader(const QByteArray &value)
     _navigationAppsEtagResponseHeader = value;
 }
 
+bool AccountState::isDesktopNotificationsAllowed() const
+{
+    return _isDesktopNotificationsAllowed;
```

Comment:
@camilasan 
Now, I am also wondering, can this value be calculated via: `_userStatus->status() == _userStatus::Status::Online`? Just curious, if we need this additional flag that stores something that we already have in the form of the enum, or not.

Reply:
this one is coming from the notification api, I think we should check them both.